.github/workflows/ci.yml

```diff
@@ -185,8 +159,8 @@ jobs:
       - name: Run csi plugin and csi-sanity test
         run: |
           export ETCD_END_POINT=127.0.0.1:2379
-          target/debug/csi --endpoint=unix://$NODE_SOCKET_FILE --workerport=0 --nodeid=localhost --drivername=io.datenlord.csi.plugin --datadir=/tmp/datenlord_data_dir --runas=node --etcd=$ETCD_END_POINT &
-          target/debug/csi --endpoint=unix://$CONTROLLER_SOCKET_FILE --workerport=0 --nodeid=localhost --drivername=io.datenlord.csi.plugin --datadir=/tmp/datenlord_data_dir --runas=controller --etcd=$ETCD_END_POINT &
+          target/debug/datenlord --endpoint=unix://$NODE_SOCKET_FILE --workerport=0 --nodeid=localhost --nodeip=127.0.0.1 --drivername=io.datenlord.csi.plugin --mountpoint=/tmp/datenlord_data_dir --runas=node --etcd=$ETCD_END_POINT --volume_info="fuse-test-bucket;http://127.0.0.1:9000;test;test1234" &
```

Comment:
Why do we still need to run two commands? Can we merge them into one?

Reply:
Yes. we need two commands. They are running as different roles. One is as csi node another is as csi controller.

Reply:
It is planned to standardize command parameters so that command is not too long.